framework/encode/vulkan_capture_manager.cpp

```diff
@@ -2822,5 +2823,34 @@ void VulkanCaptureManager::PreProcess_vkBindImageMemory2(VkDevice
     }
 }
 
+void VulkanCaptureManager::PostProcess_vkCreateShaderModule(VkResult                        result,
```

Comment:
heads up, there are 3 spots you can get SPIR-V

1. here
2. `vkCreateShaderEXT`
3. with both `VK_KHR_maintenance5` and `graphicsPipelineLibrary` you can inline things and just pass a `VkShaderModuleCreateInfo` directly into`VkPipelineShaderStageCreateInfo::pNext`

Reply:
cool, only had vkCreateShaderEXT on the radar (think those are subject to follow-up PRs)